### PR TITLE
States refactoring

### PR DESCRIPTION
**Pull request by daniel86**
The pull request does some refactoring

- move State definition to EASE-STATE.owl
- rename Gestallt to StateType

And it changes the comment of the *State* concept. Note that the previous comment was rather describing what a configuration is, so I have moved it to the *Configuration* concept.

### CODE REVIEW

**Comment by daniel86** [comment]
There is one issue: where should *hasPhase* be defined?
It refers to process and state. It is currently defined in EASE.owl.
Maybe we should rather merge together EASE-PROC and EASE-STATE?

File: owl/EASE-STATE.owl [hunk 1]
```diff
@@ -27,27 +27,27 @@
...
 
-    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#hasGestallt">
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#hasStateType">
         <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isRelatedToConcept"/>
-        <owl:inverseOf rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#isGestalltOf"/>
+        <owl:inverseOf rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#isStateTypeOf"/>
         <rdfs:domain rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
-        <rdfs:range rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#Gestallt"/>
+        <rdfs:range rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#StateType"/>
         <rdfs:comment>A relation between roles and gestallts, e.g. &apos;the chair is the supporter of the person sitting on it&apos;.</rdfs:comment>
```
Comment (mpomarlan) [inline owl/EASE-STATE.owl]: Should update comment to say, "A relation between roles and state types"

File: owl/EASE-STATE.owl [hunk 2]
```diff
@@ -27,27 +27,27 @@
...
-    <!-- http://www.ease-crc.org/ont/EASE-STATE.owl#isGestalltOf -->
+    <!-- http://www.ease-crc.org/ont/EASE-STATE.owl#isStateTypeOf -->
 
-    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#isGestalltOf">
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#isStateTypeOf">
         <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isRelatedToConcept"/>
-        <rdfs:domain rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#Gestallt"/>
+        <rdfs:domain rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#StateType"/>
         <rdfs:range rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
         <rdfs:comment>A relation between roles and gestallts, e.g. &apos;the chair is the supporter of the person sitting on it&apos;.</rdfs:comment>
```
Comment (mpomarlan) [inline owl/EASE-STATE.owl]: Should update comment to "A relation between roles and state types"

Comment (daniel86) [reply owl/EASE-STATE.owl]: done

File: owl/EASE-STATE.owl (hunk 1 above)
Comment (daniel86) [reply owl/EASE-STATE.owl]: done

**Pull request merged**